.phpstan/phpstan.neon.dist

```diff
@@ -8,6 +8,10 @@ parameters:
     excludes_analyse:
         # temporarily ignore template files
         - ../src/Resources/**.tpl.php
+        - ../src/Menu/Matcher/Voter/ActiveVoter.php
```

Comment:
I had to add those because I was getting messages like:
```
Return type mixed of method Sonata\AdminBundle\Menu\Matcher\Voter\ActiveVoter::matchItem() is not covariant with return type bool|null of method
         Knp\Menu\Matcher\Voter\VoterInterface::matchItem().
```

So I tried to ignore it and I got:

```
Error message "Return type mixed of method Sonata\AdminBundle\Menu\Matcher\Voter\ActiveVoter::matchItem() is not covariant with return type bool|null of method
     Knp\Menu\Matcher\Voter\VoterInterface::matchItem()." cannot be ignored, use excludes_analyse instead.
```